.github/workflows/simulation-1.2.yml

```diff
@@ -268,9 +265,6 @@ jobs:
     - name: Run
       run: |
         ulimit -c unlimited
-        ./script/test cert_suite ./tests/scripts/thread-cert/Cert_*.py
-        ./script/test cert_suite ./tests/scripts/thread-cert/test_*.py
-        ./script/test cert_suite ./tests/scripts/thread-cert/v1_2_*.py
         ./script/test cert_suite ./tests/scripts/thread-cert/addon_test_channel_manager_autocsl*.py
```

Comment:
We can keep the tests, but they may not provide any additional value since all these test cases are already covered by other jobs in this workflow.

The key distinction of this job is that it enables the "CSL auto-channel select" feature, which requires an explicit API call to activate. None of the existing tests enable or use this feature.

It makes sense to run only the `addon_test_channel_manager_autocsl*` tests as they are the only ones that enable and validate this specific feature.

Reply:
Maybe unrelated to this PR, but I'm wondering if we should validate that all the other basic functions continue to work when this feature is enabled.

Reply:
Ah! Sorry. I somehow missed this comment.

I'm not sure if we need to do this with `CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE` feature (under this workflow job) since its scope is limited to the "ChannelManager" module and should not affect any other core functions and also requires explicit API call for it to be enabled and used.

I don't recall the specific reason, but there was an issue preventing us from enabling CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE under other jobs (which running all tests). It might be worth revisiting this to see if we can change it so we can run the "channel manager auto csl" tests alongside other tests within the same GitHub action workflow job.

Reply:
Yeah, we should probably revisit at some point. Thanks.